README.md

```diff
@@ -1,128 +1,104 @@
-# IBM Cloud <module_name> - Terraform Module
+# Terraform Module - Cloud Pak for Security Install
 
-Mention the purpose of writing these modules.
+This Terraform Module installs **Cloud Pak for Security Operator** on an Openshift (ROKS) cluster on IBM Cloud. Once the Terraform module has run a cluster will install the CP4S operator creating the threat management resource.  After the threat management resource is created further configuration will be needed, you can follow the instructions on the CP4S documentation [here](https://www.ibm.com/docs/en/cloud-paks/cp-security/1.8?topic=security-postinstallation)
 
-E.g:
+**Module Source**: `github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/cp4s`
 
-This is a collection of modules that make it easier to provision and configure Observability services like logging, monitor and activity tracker on IBM Cloud Platform:
-* [logging-logdna](modules/logging-logdna)
-* [monitoring-sysdig](modules/monitoring-sysdig)
-* [activity-tracker-logdna](modules/activity-tracker-logdna)
+- [Terraform Module to install Cloud Pak for Security](#terraform-module-to-install-cloud-pak-for-security)
+  - [Required command line tools](#setup-tools)
+  - [Set up access to IBM Cloud](#set-up-access-to-ibm-cloud)
+  - [Provisioning this module in a Terraform Script](#provisioning-this-module-in-a-terraform-script)
+    - [Setting up the OpenShift cluster](#setting-up-the-openshift-cluster)
+    - [Using the CP4S Module](#using-the-cp4s-module)
+  - [Input Variables](#input-variables)
+  - [Executing the Terraform Script](#executing-the-terraform-script)
 
-## Compatibility
+## Setup Tools
 
-This module is meant for use with Terraform 0.13 (and higher).
+The cloud pak for security installer runs on your machine, for the installer go [here](https://www.ibm.com/docs/en/cloud-paks/cp-security/1.6.0?topic=tasks-installing-developer-tools) to be sure your command line tools are compatible.
 
-## Usage
+Terraform plugins will be Terraform 0.13 or later and terraform-provide-ibm 1.34 or later
 
-Examples are captured in [examples](./examples/) folder, demonstarte how to use a module through a template:
+## Set up access to IBM Cloud
 
-e.g:
+If running these modules from your local terminal, you need to set the credentials to access IBM Cloud.
 
-```hcl
-provider "ibm" {
-}
+Go [here](../../CREDENTIALS.md) for details.
 
-data "ibm_resource_group" "logdna" {
-  name = var.resource_group
-}
+## Provisioning this module in a Terraform Script
 
-module "logdna_instance" {
-  source  = "terraform-ibm-modules/observability/ibm//modules/logging-logdna"
+### Setting up the OpenShift cluster
 
+NOTE: An OpenShift cluster is required to install the Cloud Pak. This can be an existing cluster or can be provisioned using our `roks` Terraform module.
```

Comment:
Make the word roks to point to actual terraform module repo to provision a new openshift cluster

https://github.com/terraform-ibm-modules/terraform-ibm-cluster/tree/master/modules

Reply:
added link